.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+## Description
+
+This PR closes #{issue_number}.
+
+<!-- 
+    Write a short description to explain changes that are not mentioned in the initial issue.
+    What were the reasons for those changes?
```

Comment:
Huh having them in the code fastens the development - no need to go to the PR..

Reply:
Sorry, what do you refer to by "them" here? Do you mean, the reasons should be described in code comments/docs? Should we remove this sentence here?

Reply:
> Sorry, what do you refer to by "them" here? 

comments.

> Do you mean, the reasons should be described in code comments/docs? 

yes.

> Should we remove this sentence here?

no.